en/Editing and formatting/Obsidian Flavored Markdown.md

```diff
@@ -0,0 +1,7 @@
+Obsidian strives for maximum capability without breaking any existing formats. As a result, we use a slightly unorthodox combination of flavors of Markdown.
+
+Obsidian mainly uses [CommonMark](https://commonmark.org/), with some functionality from [GitHub Flavored Markdown](https://github.github.com/gfm/) and LaTeX.
+
+In addition, Obsidian adds syntax for [[Internal links]] (`[[ ]]`) and for [[Embedding files]] (`![[ ]]`) syntax.
```

Comment:
Comment and callout are also unique to Obsidian. Do we want to mention them for the sake of completeness? Or should it be along the lines of "adds syntax for things like Internal links and Embedding files"?

Reply:
Good catch! They should be added as well!